### DISCUSSION TITLE
What's going on with the V wrapper for raylib 4.5?

### DISCUSSION THREAD

**Discussion by Wajinn** [irishgreencitrus/raylib.v#23] [General]
There is already an [alternate branch](https://github.com/irishgreencitrus/raylib.v/tree/update_raylib_v4_5) for raylib 4.5, which includes additions such as [raygui.v](https://github.com/irishgreencitrus/raylib.v/blob/update_raylib_v4_5/raygui.v), [raymath.v](https://github.com/irishgreencitrus/raylib.v/blob/update_raylib_v4_5/raymath.v), and [rlgl.v](https://github.com/irishgreencitrus/raylib.v/blob/update_raylib_v4_5/rlgl.v).  Would appear to be exciting that this is just about available for the V community.

People would probably like to know when is it expected to become the main branch?

**Comment by irishgreencitrus** [comment]
Unfortunately, since this project is all free work as with the nature of most open source projects, and mostly mine at that, I don't have deadlines for releases unfortunately.

As for the `update_raylib_v4_5` branch specifically, I have run into a multitude of issues with even getting it to compile on my machine (windows) and I will be trying it on linux soon. Basically, it is currently not in a state where I would consider merging it into the `main` branch as that would be unfair for current users of the project.

I may need to do a complete rewrite of the 4.5 branch, which might be easier than fixing the problems, or I might wait until raylib `v5.0` is released as it currently doesn't seem that far off, looking at raylib's discord.
